app/views/layouts/application.html.slim

```diff
@@ -25,12 +25,12 @@ html lang="#{I18n.locale}"
     link rel='mask-icon' href='/safari-pinned-tab.svg' color='#e21c3d'
     meta name='theme-color' content='#ffffff'
 
-    <!--[if lt IE 10]>
```

Comment:
This seemed to pretty much be a check for IE 9 only (since IE 8 and below has its own stylesheet), so I hardcoded the version